### PR TITLE
[MPQEditor] Introduce 'media' for MPQEditor

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit introduces 'media'  for MPQEditor.

Draft: #1505
Related: #1491
ONE-vscode-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

File: media/MPQEditor/index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mixed Precision Quantization editor</title>
+    <link rel="stylesheet" href="${cssUri}">
+    <link rel="stylesheet" href="${codiconUri}">
+    <script nonce="${nonce}" type="module" src="${toolkitUri}"></script>
+    <script nonce="${nonce}" type="module" src="${scriptUri}"></script>
+</head>
+<body>
+    <div class="maintitle">
+        <span>Edit mixed precision quantization json file</span>
```
Comment (dayo09) [inline media/MPQEditor/index.html:34]: I personally think this title is not redundant. How about removing this?

Comment (stamalakhov) [reply media/MPQEditor/index.html:34]: ok.

Comment (stamalakhov) [reply media/MPQEditor/index.html]: done

**Comment by dayo09** [comment]
Thanks for your work! 😄 

Inked62e0e08f-a495-4a0c-85a4-fc13f9382fa4
BTW, when I click on that 'X' icon in 'visq path' box, it only reloads the model.circle file on the right side. What does it mean to be?

**Comment by stamalakhov** [comment]
> Thanks for your work! smile
> 
> Inked62e0e08f-a495-4a0c-85a4-fc13f9382fa4 BTW, when I click on that 'X' icon in 'visq path' box, it only reloads the model.circle file on the right side. What does it mean to be?

It clears `visq_view`.
By default, the `model-view` opens in `selector` mode, but in case `visq` data is loaded, `model_view` load the model in `visq` mode. So to reset to `selector` mode, we may press the `x` button. But IMHO in case we don't `visq` data, it should be disabled.

**Comment by dayo09** [comment]
@stamalakhov 

Oh, I think that you have removed the code. 
So it's now unrelated but to say, I would like to suggest 'Show Graph' or 'Display Graph' instead of 'Model Graph'?

**Comment by stamalakhov** [comment]
> @stamalakhov
> 
> > But IMHO in case we don't visq data, it should be disabled.
> 
> Oh, It seems that you just have removed the code. So it's now unrelated but to say, but for later, I would like to suggest 'Show Graph' or 'Display Graph' instead of 'Model Graph', for the text next to the checkbox.

Ahhh. OK. I'll rename it.

**Review by dayo09 — APPROVED** [review]
LGTM, Thanks.

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**